### PR TITLE
[ci-skip] Mention orphans in Associations

### PR DESCRIPTION
**Pull request by lambdakilo**
### Motivation / Background

This Pull Request has been created because the word "orphan" isn't mentioned in a place relative to dealing with orphans.

### Detail

This Pull Request changes guides/source/association_basics.md

### Checklist

Before submitting the PR make sure the following are checked:

* [x] This Pull Request is related to one change. Changes that are unrelated should be opened in separate PRs.
* [x] Commit message has a detailed description of what changed and why. If this PR fixes a related issue include it in the commit message. Ex: `[Fix #issue-number]`
* [x] Tests are added or updated if you fix a bug or add a feature.
* [x] CHANGELOG files are updated for the changed libraries if there is a behavior change or additional feature. Minor bug fixes and documentation changes should not be included.

### CODE REVIEW

File: guides/source/association_basics.md [hunk 1]
```diff
@@ -70,6 +70,8 @@ Deleting an author and all of its books is *much* easier:
 @author.destroy
 ```
 
+This is also knowns as dealing with orphaned data.
```
Comment (MatheusRich) [inline guides/source/association_basics.md:73]:
```suggestion
This is also known as dealing with orphaned data.
```

**Comment by fatkodima** [comment]
How this sentence is relevant to the surrounding it documentation?

**Comment by akirataguchi115** [comment]
Thanks for the comment, @fatkodima ! Let's assume that I want to know how to deal with orphans in RoR models. I go to Guides index and then AR Associations because "In Rails, an association is a connection between two Active Record models." Then I try to search for "orphan" but other than two other hits, nothing useful comes up. With this change the searching user hits the right initial spot, `has_many :books, dependent: :destroy`.

Do let me know if this motivation is wrong or not enough 🦄

**Comment by matthewd** [comment]
Yeah, thanks for the suggestion -- and I do acknowledge the guide is already a big blob of text that's hard to navigate -- but I don't think we want to pad it further by keyword stuffing terms we think people might be searching for. 

There's upcoming work planned on the documentation and its navigation, so I hope we'll find a way to make things more discoverable in that way.

**Comment by akirataguchi115** [comment]
Great to hear, @matthewd ! Thanks for closing the PR for me. Could you paste a link here to the upcoming work?

**Pull request closed without merging**